AlphaWallet/Sell/ViewControllers/EnterSellTokensCardPriceQuantityViewController.swift

```diff
@@ -94,9 +94,13 @@ class EnterSellTokensCardPriceQuantityViewController: UIViewController, TokenVer
         dollarCostLabel.translatesAutoresizingMaskIntoConstraints = false
 
         pricePerTokenField.translatesAutoresizingMaskIntoConstraints = false
-        ethPrice.subscribe { [weak self] value in
-            if let value = value {
-                self?.pricePerTokenField.ethToDollarRate = value
+        if config.chainID == 100 {
+            self.pricePerTokenField.cryptoToDollarRate = 1
```

Comment:
Looks like it's possible to get the USD price of DAI with `https://api.coinmarketcap.com/v1/ticker/dai/`. Maybe we can do it in a separate PR?